modules/ROOT/pages/tools/io-tool.adoc

```diff
@@ -0,0 +1,110 @@
+
+https://io.my-agrirouter.com/[The agrirouter InOut-Tool] is a tool to simulate a remote endpoint and to learn to communicate via agrirouter. 
+
+image::io-tool/settings.png[]
+
+
+Using the IO-Tool you can 
+
+* exchange data
+* send and receive telemetry data
+* set preferences for the data exchange
+* check the logs and debug
+* simulate end points and set routings
+
+
+== Getting Started
+
+First you need to log in to your account, which can be requested from Agrirouter support (support@my-agrirouter.com).
+
+Here is an email template for you
+
+_Dear Sir/Madam,_
+
+_My name is (first name, last name) from (company) and I would like to request an IO-Tool account for the email address (your email). In case of any questions please reach out to (contact person)._
+
+_Thank you in advance._
+
+
+image::io-tool/login.png[]
+
+The main page of the OI-Tool is then displayed with the three menu sections 
+
+* Data Storage
+* Logging
+* Settings
+
+If you only see these three, it means that you are *not* yet connected to the agrirouter.
+
+== Data Storage
+
+This view allows you to select the data you intend to transfer.
+
+image::io-tool/datastorage.png[]
+
+You can either upload files from your device or select from the list of previously uploaded files. The system will 
+automatically select the file type, which can be manually adjusted later. Files are grouped by type in expandable sections.
+
+//TODO-Frank please provide some data I can upload and test (successful, unsuccessful)
+
+The lower panel contains the logs of all actions, which can also be expanded.
+
+== Logging
+
+The Logging section of the Main Menu provides more detailed information about the actions performed. 
+
+image::io-tool/logging.png[]
+
+// upload a screenshot later when more actions are performed
+
+== Getting Ready for Data Exchange
+
+To be able to use the IO-Tool for data exchange, you need to connect the IO-Tool to an agrirouter account. To do this, go to the Settings section in the main menu and select either MQTT or REST in the Production or QA (Quality Assurance) environment for the agrirouter connection. QA refers to the quality assurance of the agrirouter, not your application.
+
+When deciding between QA or Production, consider that new applications are only created in Production. Normally you would want to use MQTT because it is faster. Here you can find more information about https://docs.agrirouter.com/agrirouter-interface-documentation/latest/communication.html#mqtt[MQTT] and https://docs.agrirouter.com/agrirouter-interface-documentation/latest/communication.html#rest[REST].
+
+After selecting the protocol you will be taken to the *agrirouter login* view and need to confirm the action in this screen:
+
+(_Please note that after disconnecting from the agrirouter account you will need to wait for ca. 5 minutes to be able to connect again.`_)
+
+image::io-tool/connect.png[]
+
+Please be patient as this may take a while to get connected.
+
```

Comment:
A remark Box here could refer to the Authorization Process like "At this point you can see the Authorization and Onboarding Process from an end users perspective."
And in that, link the description of the Process